amethyst_renderer/src/pass/shaders/fragment/flat_color.glsl

```diff
@@ -0,0 +1,16 @@
+// TODO: Needs documentation.
```

Comment:
Are you gonna add the documentation for this and the vertex shader in this PR? It's not too much code, so I don't think too much documentation is needed.

Reply:
Removed

Reply:
Thanks!